read-fonts/src/tables/cbdt.rs

```diff
@@ -0,0 +1,124 @@
+//! The [CBDT (Color Bitmap Data)](https://docs.microsoft.com/en-us/typography/opentype/spec/cbdt) table
+
+use super::bitmap::{BitmapData, BitmapLocation};
+
+include!("../../generated/generated_cbdt.rs");
+
+impl<'a> Cbdt<'a> {
+    pub fn data(&self, location: &BitmapLocation) -> Result<BitmapData<'a>, ReadError> {
+        super::bitmap::bitmap_data(self.offset_data(), location, true)
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::super::bitmap::{BitmapDataFormat, SmallGlyphMetrics};
+    use crate::{types::GlyphId, FontRef, TableProvider};
+
+    #[test]
+    fn read_cblc_1_cbdt_17() {
+        let font = FontRef::new(font_test_data::EMBEDDED_BITMAPS).unwrap();
+        let cblc = font.cblc().unwrap();
+        let cbdt = font.cbdt().unwrap();
+        let size = &cblc.bitmap_sizes()[0];
+        // Metrics for size at index 0
+        assert_eq!(size.hori.ascender(), 101);
+        assert_eq!(size.hori.descender(), -27);
+        assert_eq!(size.hori.width_max(), 136);
+        assert_eq!(size.vert.ascender(), 101);
+        assert_eq!(size.vert.descender(), -27);
+        assert_eq!(size.vert.width_max(), 136);
```

Comment:
Optional, I prefer the half dozen or so asserts here collapsed to one

Reply:
Done